llvm/lib/MC/RepoObjectWriter.cpp

```diff
@@ -100,15 +103,15 @@ class RepoObjectWriter : public MCObjectWriter {
     /// dependent.
     SmallVector<pstore::typed_address<pstore::repo::compilation_member>, 4>
         Dependents;
+    // A pointer to the corresponding compilation member in the
+    // CompilationMembers.
+    pstore::repo::compilation_member *CorrespondingCompilationMember = nullptr;
```

Comment:
Since this points into CompilationMembers, could it be an iterator rather than a raw pointer?

Reply:
Yes, it has been changed to an iterator.